docs/user-guide/station-device-provisioning.md

```diff
@@ -158,13 +160,101 @@ If you don't want to use the LoRa Device Provisioning CLI, in the following sect
         }
         ```
 
+        === "CN470RP1 Example Configuration"
+
+        ``` json
```

Comment:
Let's maybe add a comment saying that enabled channels for this configuration are 1-4 and 40-43?